swagger.js

```diff
@@ -328,8 +328,7 @@ export default {
         description: 'Returns an authentication token on success.',
```

Comment:
you need to add documentation for these endpoints

Reply:
@seyi-adeleke Yes, I am working on that.